### DISCUSSION TITLE
First time Use...I'm confused.

### DISCUSSION THREAD

**Discussion by chidinweke** [neonbjb/tortoise-tts#701] [General]
I successfully installed Tortoise-TTS on my localhost. Then I did a system restart.  
I'm confused on how to start Tortoise-TTS on my localhost for the first time.

Please I need help. I use a Windows PC.

**Comment by JMcCoding** [comment]
This post is super vague, so I'm not really surprised nobody's responded.  All you said was you installed it onto your localhost/Windows PC.  Hope this helps:

- Have you checked the [README](https://github.com/neonbjb/tortoise-tts/blob/main/README.md) for the usage examples?  There are several entries explaining how to use the provided python files (Such as `do_tts.py`)
- If the above doesn't help,  you'll probably have to clarify things such as:
  1. What steps did you take for installation?
  2. What type of installation did you use?
     - There are several ways, such as Windows Subsystem for Linux (WSL), Docker, Conda local installation, etc
  3. What are your specifications for the installed system?
     - IE: GeForce GTX 1650 Ti graphics card, Cuda 12.1, Windows 11, etc
  4. Are you getting to a point where you have an error?
     - If so, I recommend from my time trying to get it running that you search the repo for the error messages.  There are a ton of issues posted already, and I found several of them helpful.

Just dropping by as a fellow Windows user who got it to work at least once.
